frontend-public/src/components/dashboard/mineInfo/MineInfoList.js

```diff
@@ -62,9 +62,7 @@ const columns = [
   {
     title: "Received Date",
     dataIndex: "receivedDate",
-    render: (text, record) => (
-      <div title="Received Date">{record.doc.received_date || "-"}</div>
-    ),
+    render: (text, record) => <div title="Received Date">{record.doc.received_date || "-"}</div>,
```

Comment:
I'm fairly sure our autoformatting will always wrap JSX from an implicit return in parentheses. Did you intentionally make this change? I'm curious why it would happen otherwise

Reply:
No, I just saved the file and pushed it. This was auto formatting. I only modified the document section explicitly.

Reply:
Very odd. The other changes look good, so I'll approve. I don't like that your formatted did something different. We should keep an eye on this to make sure you're not going to end up with big diffs in the future based strictly on your formatting being different. Everyone's should be exactly the same automatically.

Reply:
is there a different rule-set applied to the public site?